yoyodyne/train.py

```diff
@@ -141,7 +148,6 @@ def main(
     label_smoothing,
     gradient_clip,
     batch_size,
-    eval_batch_size,
```

Comment:
Removing an `eval_batch_size` is going to make training runs slower. Some of our models evaluate at about 2-5x the size of training. I think the extra variable is worthwhile.

Reply:
Why don't we set it to 2x batch size? (Or default it to that?) Thoughts?

Reply:
Setting default as 2x sounds safe. Should allow best of both worlds.